### ISSUE TITLE
Consider keeping a changelog and publishing git tags for releases

### ISSUE THREAD

**Issue by shepmaster** [liady/webpack-node-externals#81]
2.5.1 appears to have been released, but it's difficult to see exactly what changed without either of these artifacts. It *appears* to be [this diff](https://github.com/liady/webpack-node-externals/compare/23c124af73388ade75e3177e3a845e62a7c2698b...b21c8c05da8c7ef02f1f81469a2fcc93a8425c69).

**Comment by aneurysmjs** [comment]
yeah. maybe you can use [https://github.com/commitizen/cz-conventional-changelog](url) and [https://github.com/commitizen/cz-cli](url)

**Comment by natterstefan** [comment]
Hi @liady and other maintainers,

please consider adding a changelog. This makes it easier for users to handle breaking and other changes. Thanks a lot.
